ods_ci/tests/Resources/OCP.resource

```diff
@@ -126,7 +126,7 @@ Wait For Pods To Be Ready
 
 Wait For Pods To Succeed
     [Arguments]    ${label_selector}    ${namespace}    ${timeout}=300s    ${exp_replicas}=${NONE}
-    Wait Until Keyword Succeeds    ${timeout}    3s
+    Wait Until Keyword Succeeds    ${timeout}    15s
```

Comment:
better to parametrize it. The retry depends on the timeout as well, this may impact other test cases